src/main/java/org/mitre/synthea/export/FhirR4.java

```diff
@@ -1,18 +1,28 @@
 package org.mitre.synthea.export;
 
+// Add chargeItem resource
```

Comment:
The comment is not needed, consider removing it. The package is imported so the CahrgeITem is added and comment does not carry extra meaning.